inc/functions/files.php

```diff
@@ -854,7 +854,12 @@ function rocket_clean_domain( $lang = '' ) {
 		do_action( 'before_rocket_clean_domain', $root, $lang, $url ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
 
 		try {
-			$files = new RegexIterator( $iterator, "/{$file['host']}*{$file['path']}/i" );
+			$regex = sprintf( "/{$file['host']}*%s/i",
+				empty( $file['path'] )
```

Comment:
Shouldn’t the logic be reversed?
```suggestion
				'' !== trim( $file['path'], '/\\' )
```